Aztec/Classes/NSAttributedString/Conversions/AttributedStringSerializer.swift

```diff
@@ -108,9 +108,25 @@ class AttributedStringSerializer {
         if let representation = implicitRepresentation(for: element, inheriting: childAttributes) {
             content.append(representation)
         } else {
-            for child in element.children {
-                let childContent = serialize(child, inheriting: childAttributes)
-                content.append(childContent)
+            // if the element is of type '.a' and contains a '.img'
+            if element.isNodeType(.a) && element.children.count == 1 {
+                if let imgElement = element.children.first as? ElementNode, imgElement.isNodeType(.img) {
+
+                    // get the link URL and assign it to the image attachment
+                    if let linkText = element.stringValueForAttribute(named: "href") {
```

Comment:
Logic: If this is false, we're losing content.

We should actually set `linkText` to `""` when `href` is missing.